### ISSUE TITLE
Adjust robot.zip for shorter matches

### ISSUE THREAD

**Issue by PeterJCLaw** [srobo/tasks#270]
For SR2019, our matches are going to be 2:30 rather than the usual 3 minutes. Since we rely on the robots turning themselves off at the end of matches, we need to fix this before the competition starts.

**Comment by PeterJCLaw** [comment]
`zip-packager` branch [`sr2019-shorter-matches`](https://www.studentrobotics.org/cgit/brain/zip-packager.git/log/?h=sr2019-shorter-matches) exists with a likely fix. I'm not in a position to test it, however I've created a zip which can be used for that purpose.

Attached is a `robot.zip` which will beep until stopped while also printing the time since the start button was pressed to the log.
The code is saved as the "duration" project of team "ABC" in the IDE.
Importantly, the attached version zip was built using the fix branch (linked above) and thus should be killed after 2:30 rather than three minutes.
We need to test this a reference zip (https://studentrobotics.org/ide/zips/ABC/duration/6c64843/robot.zip; to check that the code works as expected) and then also this zip on a kit before we deploy the fix.

I've also emitted some [notes](https://gist.github.com/PeterJCLaw/287c9781bdd0ad3d0645809366637443) from when I first did some squashfs munging, though this time I used the simpler route described in https://unix.stackexchange.com/questions/80305/mounting-a-squashfs-filesystem-in-read-write.
